projects/canopy/src/lib/button/button.interface.ts

```diff
@@ -1,10 +1,11 @@
-export type Variant =
+export type ButtonVariant =
```

Comment:
My personal preference would be to have the types without the `Button` prefix since they all come from `button.interface.ts` but not too fussed about it.

Reply:
My reason for this change was the chance of a naming clash in certain composite components. An example scenario could be a component that needs an `AlertVariant` and a `ButtonVariant`. This change does also bring a bit of consistency with the other interfaces which all have prefixes.

Reply:
Yes, makes sense 👍